scripts/build.sh

```diff
@@ -43,7 +43,7 @@ echo "==> Building..."
 gox \
     -os="${XC_OS}" \
     -arch="${XC_ARCH}" \
-    -ldflags "-X main.GitCommit ${GIT_COMMIT}${GIT_DIRTY}" \
+    -ldflags "-X=main.GitCommit ${GIT_COMMIT}${GIT_DIRTY}" \
```

Comment:
This looks wrong, shouldn't this be instead:
`-X main.GitCommit=${GIT_COMMIT}${GIT_DIRTY}`
Notice the position of the "="